.github/workflows/full-tests.yml

```
@@ -49,6 +49,18 @@ jobs:
           -Dquarkus.container-image.build=true \
           -Dquarkus.container-image.push=false
 
+      - name: Build csv-payments monolith runtime
+        run: |
+          ./examples/csv-payments/build-monolith.sh -DskipTests
+
+      - name: Run csv-payments monolith E2E
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:1:8: Double quote to prevent globbing and word splitting

(shellcheck)